pkg/datarepair/checker/checker.go

```diff
@@ -55,10 +56,11 @@ type Checker struct {
 	Loop            sync2.Cycle
 	IrreparableLoop sync2.Cycle
 	monStats        durabilityStats
+	Garbage         *gc.Garbage
```

Comment:
Calling this Garbage looks really weird, also order of fields

Reply:
😄 I agree, it's a placeholder for now. I need a word besides queue since it's not a queue! open to suggestions. I'll fix the order of fields.